src/CMakeLists.txt

```diff
@@ -60,14 +60,17 @@ set(LIBRARY_PUBLIC_HEADERS
     mococrw/symmetric_crypto.h
     mococrw/subject_key_identifier.h
     mococrw/util.h
-    mococrw/util.h
     mococrw/x509.h
 )
 
+set(LIBRARY_PRIVATE_HEADER_DIRS
+    mococrw/private/
+)
+
 
 add_library(${LIBRARY_NAME} SHARED ${LIBRARY_SOURCES} ${LIBRARY_PUBLIC_HEADERS})
 add_library(${PROJECT_NAME}::${LIBRARY_NAME} ALIAS ${LIBRARY_NAME})
-target_include_directories(${LIBRARY_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/cwrap")
+target_include_directories(${LIBRARY_NAME} PRIVATE ${LIBRARY_PRIVATE_HEADER_DIRS})
```

Comment:
Is this even needed if you're including the headers with their relative path `mococrw/private` anyway? I don't think it is, in which case this should be removed.